qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -147,19 +157,26 @@ class SymbolicPulse(Pulse):
     .. rubric:: Envelope function
 
     This is defined with an instance attribute :attr:`SymbolicPulse.envelope`
-    which can be provided through its setter method.
-    The expression must be a function of ``t``, ``duration``, ``amp`` and
-    any additional parameters specified for the pulse shape to implement.
-    The time ``t`` and ``duration`` are in units of dt, i.e. sample time resolution,
-    and this function is sampled with a discrete time vector in [0, ``duration``]
-    sampling the pulse envelope at every 0.5 dt (middle sampling strategy) when
-    :meth:`SymbolicPulse.get_waveform` method is called.
-    The ``amp`` is a complex-valued coefficient that scales the symbolic pulse envelope.
-    This indicates conventionally the Qiskit Pulse conforms to the IQ format rather
-    than the phasor representation. When a real value is assigned to the ``amp``,
+    which can be provided through the symbolic pulse constructor.
+    The envelope function at time :math:`t` must be defined in the form
```

Comment:
I think it is a little bit confusing that both F and F' are referred to as the envelope and one of them corresponds to the `envelope` attribute. Can we differentiate between a little more between F and F'?

Reply:
Added more details.